model/jpa/src/main/java/org/keycloak/organization/jpa/JpaOrganizationProvider.java

```diff
@@ -191,6 +191,7 @@ private boolean addMember(OrganizationModel organization, UserModel user, Member
             }
 
             user.joinGroup(group, metadata);
+            OrganizationModel.OrganizationMemberJoinEvent.fire(organization, user, session);
```

Comment:
what exactly is the purpose of having this OrganizationMemberJoinEvent interface and firing it here?